### PR TITLE
[Paris-2018] add stub

### PR DESCRIPTION
**Pull request by phrawzty**
Add stub for Paris 2018 event. 🇫🇷 

`r?` @jooooooon

### CODE REVIEW

File: content/events/2018-paris/location.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
++++
+Title = "Location"
+Type = "event"
+Description = "Location for devopsdays Paris 2018"
++++
+
+Watch this space for information about the venue including address, map/direction, parking/transit, and any hotel details.
```
Comment (jooooooon) [inline content/events/2018-paris/location.md]: Can we already put the Grand Rex in here?

File: content/events/2018-paris/propose.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
++++
+Title = "Propose"
+Type = "event"
+Description = "Propose a talk for devopsdays Paris 2018"
++++
+  {{< cfp_dates >}}
+
+<hr>
+
+There are three ways to propose a topic at devopsdays:
```
Comment (jooooooon) [inline content/events/2018-paris/propose.md]: Could we take this from https://www.devopsrex.fr/proposer-un-talk/?

File: content/events/2018-paris/sponsor.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Title = "Sponsor"
+Type = "event"
+Description = "Sponsor devopsdays Paris 2018"
++++
+
+We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
+
+<hr>
+
+devopsdays is a self-organizing conference for practitioners that depends on sponsorships. We do not have vendor booths, sell product presentations, or distribute attendee contact lists. Sponsors have the opportunity to have short elevator pitches during the program and will get recognition on the website and social media before, during and after the event. Sponsors are encouraged to represent themselves by actively participating and engaging with the attendees as peers. Any attendee also has the opportunity to demo products/projects as part of an open space session.
```
Comment (jooooooon) [inline content/events/2018-paris/sponsor.md]: Yes for everything except the opportunity to demo products/projects as part of an open space session which we don't have.

File: content/events/2018-paris/sponsor.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+<p>
+Gold sponsors get a full table and Silver sponsors a shared table where they can interact with those interested to come visit during breaks. All attendees are welcome to propose any subject they want during the open spaces, but this is a community-focused conference, so heavy marketing will probably work against you when trying to make a good impression on the attendees.
```
Comment (jooooooon) [inline content/events/2018-paris/sponsor.md]: This is not applicable to us however - sponsorships are somewhat more complicated than that, and no open spaces so...

File: data/events/2018-paris.yml [hunk 5]
```diff
@@ -0,0 +1,81 @@
+name: "2018-paris" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2018" # The year of the event. Make sure it is in quotes.
+city: "Paris" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsrex" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devops is alive and well in the city of lights!" # Edit this to suit your preferences
```
Comment (jooooooon) [inline data/events/2018-paris.yml]: We usually use "La conférence devops 100% retour d’expérience" but if you need English, that's fine.

File: data/events/2018-paris.yml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Jonathan Clarke"
+    employer: "Normation"
+    twitter: "jooooooon42"
+    bio: ""
+  - name: "Dan Maher"
+    twitter: "phrawzty"
+    bio: ""
+organizer_email: "organizers-paris-2018@devopsdays.org" # Put your organizer email address here
```
Comment (jooooooon) [inline data/events/2018-paris.yml]: Does this exist? Or will it? :)

File: data/events/2018-paris.yml [hunk 7]
```diff
@@ -0,0 +1,81 @@
+name: "2018-paris" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2018" # The year of the event. Make sure it is in quotes.
+city: "Paris" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsrex" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devops is alive and well in the city of lights!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
```
Comment (jooooooon) [inline data/events/2018-paris.yml]: Oooh, fancy! This is new ❤️ 
We do actually:  UA-81620980-1

**Comment by bridgetkromhout** [comment]
Thanks, @jooooooon! I would very much like to have this all properly in French as much as we can. 💯

File: data/events/2018-paris.yml (hunk 6 above)
Comment (phrawzty) [reply data/events/2018-paris.yml]: It does! :)

**Comment by phrawzty** [comment]
> but it maybe they're not actually displayed 

Yep - this stub only activates the `contact` page formally. The links to `location` and `sponsor` go to the official website.

Again, this is just the stub to get our date locked in. We'll flesh it out as we go. :)

**Pull request merged**